app/uk/gov/hmrc/leakdetection/config/ConfigLoader.scala

```diff
@@ -75,11 +74,6 @@ object Rule {
   }
 }
 
-final case class Github(
-  url: String,
-  apiUrl: String
-)
-
 final case class GithubSecrets(
```

Comment:
Do we still need GithubSecrets? Do we still need the `AdminController.checkGithubRateLimits`? (If we do, if we add the credential to teams-and-repos, it will send the rate limits to grafana)

Reply:
Yeah we still need the secrets to be able to download the zip.
I'm not sure what (if anything) calls the rate limit endpoint.  this uses `platops-general-X` so we'll probably find teams-and-repos is already tracking it.

Reply:
yes, teams-and-repos is already tracking the general token. I think it's safe to remove the rate limit endpoint.

Reply:
grafana is already monitoring the rate limits on the [leak-detection page](https://grafana.tools.production.tax.service.gov.uk/d/Qre_5og7z/leak-detection?orgId=1&refresh=15m)  
![image](https://user-images.githubusercontent.com/65957825/155313366-6e353bb2-5cc6-4eea-877e-d6d5e032b3d8.png)